corehq/apps/hqcase/utils.py

```diff
@@ -272,7 +272,7 @@ def update_case(domain, case_id, case_properties=None, close=False):
     context = {
         'case_id': case_id,
         'date_modified': json_format_datetime(datetime.datetime.utcnow()),
-        'user_id': '',
+        'user_id': 'system',
```

Comment:
as long as a case has an `owner_id` it should be fine to leave it blank, though "system" seems better. if we make this change we should also add "system" to `WEIRD_USER_IDS`.

This isn't used to make new cases, just update existing ones, right? in that case they should all have owner ids